VirtusizeCore/Sources/API/APIService.swift

```diff
@@ -57,7 +57,11 @@ open class APIService {
 		error errorHandler: ErrorHandler? = nil
 	) {
 		let task: URLSessionDataTask
+        
 		task = APIService.session.dataTask(with: request) { (data, response, error) in
+            
+            print("request :\(request.url?.absoluteString ?? "")")
+            print("error \(error.debugDescription)")
```

Comment:
Do we need those print here ?
If it is for debugging OK but not for production code.
On a side node, we should handle error correctly using try catch.

Reply:
removed print this was for debug